docs/development.md

```diff
@@ -1,28 +1,34 @@
 # Development
 
-Add details here to give a brief overview of how to work with the provider APIs.
-Please reference any SDKs or API docs used to help build the integration here.
+The Signal Science API docs can be found
+[here](https://docs.fastly.com/signalsciences/api/). Additional developer guides
+can be found [here](https://docs.fastly.com/signalsciences/developer/).
 
-## Prerequisites
+## Provider account setup
 
-Supply details about software or tooling (like maybe Docker or Terraform) that
-is needed for development here.
+In order to use the Signal Science API, you will need to have a user account.
+Once invited, click on `My Profile` at the top right of page. Click
+[API Access Tokens](https://dashboard.signalsciences.net/corps/jupiterone/user/apitokens)
+from the drop down.
 
-Please supply references to documentation that details how to install those
-dependencies here.
+Click
+[Add API access token](https://dashboard.signalsciences.net/corps/jupiterone/user/apitokens#add)
```

Comment:
Are there specific permissions that a user needs to hit these particular APIs? 

Are we recommending that someone setting up this integration should use their _own_ user to generate an API token or to provision a readonly user and use _that_ API token?

Reply:
Will update/clarify